benchmark/all.py

```diff
@@ -0,0 +1,57 @@
+"""
+Run all configurations of the benchmarks
+"""
+
+import etcd
+import logging
+from typing import List
+import common
+import ycsb
+import argparse
+
+logging.basicConfig(format="%(asctime)s %(levelname)s %(message)s", level=logging.DEBUG)
```

Comment:
FYI, we use [loguru](https://github.com/Delgan/loguru) for logging in the CCF Python infra and have been happy with it in case you wanted to try it.

Reply:
Will take a look for another PR, thanks!